bootloader/src/old_machine_bootloader_windows.c

```diff
@@ -340,37 +341,37 @@ int ping_island(int argc, char * argv[]) {
     wchar_t* requestContentsW;
     if (server_i != 0) {
         char * server = replaceSubstringOnce(argv[server_i], ISLAND_SERVER_PORT, "");
-		size_t server_length_cb = strlen(server) + 1;
+        size_t server_length_cb = strlen(server) + 1;
         serverW = (wchar_t*)malloc(sizeof(wchar_t) * (server_length_cb));
         if (NULL == serverW) {
             free(server);
             error("Memory allocation failed\n");
         }
 
         errorValue = mbstowcs_s(NULL, serverW, server_length_cb, server, server_length_cb);
```

Comment:
Calculating serverW is relevant to both branches, please hoist and reduce duplication

Reply:
What branches are you talking about?

Reply:
Server only or server+ tunnel.

You can solve it either by hoisting the logic outside (since it will anyway happen) or initialize serverW to NULL, to prevent garbage data in the stack from making it not NULL (which is what will happen).